docs/sources/alerting/configure-notifications/create-silence.md

```diff
@@ -67,13 +68,13 @@ Silences stop notifications from being created for a specified time window but d
 Silences are assigned to a [specific Alertmanager](ref:alertmanager-architecture) and only suppress notifications for alerts managed by that Alertmanager.
 {{< /admonition >}}
 
-## Mute timings vs silences
+## Mute and active timings vs silences
 
-[Mute timings](ref:shared-mute-timings) and [silences](ref:shared-silences) are distinct methods to suppress notifications. They do not prevent alert rules from being evaluated or stop alert instances from appearing in the user interface; they only prevent notifications from being created.
+[Mute and active timings](ref:shared-mute-timings) and [silences](ref:shared-silences) are distinct methods to suppress notifications. They do not prevent alert rules from being evaluated or stop alert instances from appearing in the user interface; they only prevent notifications from being created.
 
-The following table highlights the key differences between mute timings and silences.
+The following table highlights the key differences between mute and active timings, and silences.
 
-|            | Mute timing                                                 | Silence                                                          |
+|            | Mute and active timing                                      | Silence                                                          |
```

Comment:
I think we can omit active intervals from this comparison because comparing silences with active intervals is not correct. Semantically, mute timings and silences are much closer.